src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise.component.html

```diff
@@ -2,7 +2,9 @@
     <h4 class="d-flex mb-3 flex-wrap justify-content-end">
         <span class="mb-1 me-auto">
             <span *ngIf="course && !showHeading">{{ course.title }} - </span>
-            <span *ngIf="fileUploadExercises && showHeading && !filter.isEmpty()">{{ getAmountOfExercisesString(filteredFileUploadExercises) }} / </span>
+            <span *ngIf="fileUploadExercises && showHeading && !filter.isEmpty() && fileUploadExercises.length"
+                >{{ getAmountOfExercisesString(filteredFileUploadExercises) }} /
```

Comment:
```suggestion
            <span *ngIf="fileUploadExercises && showHeading && !filter.isEmpty() && fileUploadExercises.length">
                {{ getAmountOfExercisesString(filteredFileUploadExercises) }} /
```
Formatting, or is it required like that to avoid having an additional space displayed?

Reply:
I locally had this in one line, the automatic formatter changed it to this. I cannot find a difference using this way and I also like this more.